chart/validator/templates/cleanup.yaml

```diff
@@ -86,23 +86,32 @@ metadata:
     "helm.sh/hook": pre-delete
 spec:
   template:
+    metadata:
+      labels:
+        app: validator-cleanup-job
     spec:
       restartPolicy: Never
       serviceAccountName: spectro-cleanup
       containers:
       - name: validator-cleanup
-        image: gcr.io/spectro-images-public/release/spectro-cleanup:1.1.0
+        image: gcr.io/spectro-images-public/release/spectro-cleanup:1.2.0
```

Comment:
Please expose the cleanup image as a config option. It needs to be configurable in airgapped scenarios.